app/helpers/url_helper.rb

```diff
@@ -18,10 +18,15 @@ def github_compare_to_master(application, deploy)
   end
 
   def govuk_domain_suffix(environment, on_aws:)
-    return "blue.#{environment}.govuk.digital" if on_aws
-    return "publishing.service.gov.uk" if environment == "production"
-
-    "#{environment}.publishing.service.gov.uk"
+    if environment == "integration"
+      "integration.publishing.service.gov.uk"
+    elsif on_aws
+      "blue.#{environment}.govuk.digital"
+    elsif environment == "production"
+      "publishing.service.gov.uk"
```

Comment:
Do we need these anymore? I think the only thing that's relevant here is "backdrop", but's all kinds of broken and shouldn't be a constraint. Perhaps we can just have a simple "integration" vs. "blue." conditional now, and drop `on_aws`?

Reply:
Ha, that's the can of worms I was avoiding by just adding in the fix for integration 😁 I wasn't sure if these were still being used somewhere.

But I think you're right, it can probably be simplified to that one conditional. I'll append a refactor commit.

Reply:
Nope, this was a rabbit hole I did not want to go down...! Here's a branch I've created to show how unwieldy this gets:
https://github.com/alphagov/release/compare/fix-smokey-url-integration-spike

I got it mostly working but then I need to account for whether or not there should be `blue.` in the prefix of the domain, and that looks like it would require a fair bit of rewriting all over the application. I think it's best to just apply this simple integration Smokey link fix.

Reply:
Thanks for giving it a go. It doesn't look like it's _too_ much work to properly strim it down. I suspect that it'll lie here forevermore, but then this repo has a fair few other maintenance issues!